spinel/stream.py

```diff
@@ -63,13 +63,13 @@ def __init__(self, dev, baudrate=115200):
     def write(self, data):
         self.serial.write(data)
         if CONFIG.DEBUG_STREAM_TX:
-            logging.debug("TX Raw: " + str(map(spinel.util.hexify_chr, data)))
+            logging.debug("TX Raw: " + str(list(map(spinel.util.hexify_chr, data))))
 
     def read(self, size=1):
         pkt = self.serial.read(size)
         if CONFIG.DEBUG_STREAM_RX:
-            logging.debug("RX Raw: " + str(map(spinel.util.hexify_chr, pkt)))
-        return map(ord, pkt)[0]
+            logging.debug("RX Raw: " + str(list(map(spinel.util.hexify_chr, pkt))))
+        return list(map(ord, str(pkt)))[0]
```

Comment:
Unfortunately i can't connect to my board using UART and pyspinel in Python3. I found that this function returns 126 (HDLC_FLAG) in Python2 but 98 in Python3. Because of that this [check](https://github.com/openthread/pyspinel/blob/master/spinel/hdlc.py#L82) fails.

Reply:
Thanks for catching that. I pushed an update. Let me know if that helps.